README.md

```diff
@@ -35,3 +35,14 @@ const Component: FunctionComponent = () => {
   )
 }
 ```
+
+### Options
+
+- `migrationFrom`: Adds additional CSS handles for cases where a component is migrating from another app.
+  
+#### Usage:
+```tsx
+const CSS_HANDLES = ['container']
+const handles = useCssHandles(CSS_HANDLES, { migrationFrom: 'vtex.store-components@2.x' })
+// Returns { container: 'vtex-current-app-0-x-container vtex-store-components-2-x-container' }
```

Comment:
⛏ Small detail, but the latest major of `store-components` is 3.x, so it might be good to use `vtex.store-components@3.x` here :)

Reply:
I would also delete this README file and keep just the one inside of the `/docs` folder, so that it's easier to maintain.
GitHub is pretty smart about this, and if you don't have a README file at the root of your repo, it will use the one inside `/docs`